components/brave_component_updater/browser/switches.h

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_COMPONENT_UPDATER_BROWSER_SWITCHES_H_
+#define BRAVE_COMPONENTS_BRAVE_COMPONENT_UPDATER_BROWSER_SWITCHES_H_
+
+namespace switches {
+
+extern const char kUseGoUpdateDev[];
```

Comment:
it could be `constexpr char kUseGoUpdateDev[] = " .."`, this way `.cc` is not needed